web/app/ui/DetailsDrawer.js

```diff
@@ -53,7 +53,7 @@ const DetailsDrawer = props => {
       const propsData = Object.keys(drawerData.properties).map(item => {
         return {
           key: item,
-          value: drawerData.properties[item]
+          value: JSON.stringify(drawerData.properties[item]).replace(/^"|"$/g, '')
```

Comment:
What's the meaning of `replace(/^"|"$/g, '')`?

Reply:
The JSON.stringify method will return a string wrapped in double quotes.

```
JSON.stringify('abc')
# "abc"
```